vispy/gloo/glir.py

```diff
@@ -233,7 +233,7 @@ def convert_shaders(convert, shaders):
                 lines.append(line.rstrip())
             # Write
             if not has_version:
-                lines.insert(0, '#version 120\n#line 2\n')
```

Comment:
I seem to recall having fixed this once as well. I think the various OpenGL implementations represent their line numbers differently.